sdk/monitor/azure-monitor-opentelemetry/README.md

```diff
@@ -9,21 +9,24 @@ This distro automatically installs the following libraries:
-| [OpenTelemetry Requests Instrumentation][ot_instrumentation_requests] | [requests][pypi_requests] | [link][ot_instrumentation_requests_version]
-| [OpenTelemetry UrlLib Instrumentation][ot_instrumentation_urllib] | [urllib][pypi_urllib] | All
-| [OpenTelemetry UrlLib3 Instrumentation][ot_instrumentation_urllib3] | [urllib3][pypi_urllib3] | [link][ot_instrumentation_urllib3_version]
+OpenTelemetry instrumentations allow automatic collection of requests sent from underlying instrumented libraries. The following is a list of OpenTelemetry instrumentations that come bundled in with the Azure monitor distro. These instrumentations are enabled by default. See the [Usage](#usage) section below for how to opt-out of these instrumentations.
+
+| Instrumentation | Supported library | Entry Point Name | Supported versions |
+| --------------- | ----------------- | ---------------- | ------------------ |
+| [Azure Core Tracing OpenTelemetry][azure_core_tracing_opentelemetry_plugin] | `Azure Core libraries` | `azure_sdk` | |
+| [OpenTelemetry Django Instrumentation][ot_instrumentation_django] | [django][pypi_django] | `django` | [link][ot_instrumentation_django_version]
+| [OpenTelemetry FastApi Instrumentation][ot_instrumentation_fastapi] | [fastapi][pypi_fastapi] | `fastapi` | [link][ot_instrumentation_fastapi_version]
+| [OpenTelemetry Flask Instrumentation][ot_instrumentation_flask] | [flask][pypi_flask] | `flask` | [link][ot_instrumentation_flask_version]
+| [OpenTelemetry Psycopg2 Instrumentation][ot_instrumentation_psycopg2] | [psycopg2][pypi_psycopg2] | `psycopg2` | [link][ot_instrumentation_psycopg2_version]
+| [OpenTelemetry Requests Instrumentation][ot_instrumentation_requests] | [requests][pypi_requests] | `requests` | [link][ot_instrumentation_requests_version]
+| [OpenTelemetry UrlLib Instrumentation][ot_instrumentation_urllib] | [urllib][pypi_urllib] | `urllib` | | All
+| [OpenTelemetry UrlLib3 Instrumentation][ot_instrumentation_urllib3] | [urllib3][pypi_urllib3] | `urllib3` | [link][ot_instrumentation_urllib3_version]
+
+If you would like to add support for another OpenTelemetry instrumentation, please submit a feature [request][distro_feature_request]. In the meantime, you can use the OpenTelemetry instrumentation manually via it's own APIs (i.e. `instrument()`) in your code. See [this][samples_manual] for an example.
 
 ## Azure Core Distributed Tracing
 
-Using the [Azure Core Tracing OpenTelemetry][azure_core_tracing_opentelemetry_plugin] library, you can automatically capture the distributed tracing from Azure Core libraries. See the associated [sample][azure_core_tracing_opentelemetry_plugin_sample] for more information. This feature is enabled automatically.
+Using the [Azure Core Tracing OpenTelemetry][azure_core_tracing_opentelemetry_plugin] library, you can automatically capture the distributed tracing from Azure Core libraries. See the associated [sample][azure_core_tracing_opentelemetry_plugin_sample] for more information.
```

Comment:
I guess we could hold off as long as we enable it automatically and DON'T allow a way for customers to disable it. That way, if we chose to add it to instrumentations like Node does, it wouldn't break. However, because Node is doing this and it sort of acts the same, I think it would be simpler to include it in terms of use. For documentation, though, I see what you're saying.

Reply:
Offline decision: keep it in instrumentations but remove the separate section about azure core

Reply:
Done.